src/Sarif/CodeGenHints.json

```diff
@@ -805,7 +805,7 @@
       "arguments": {
         "namespaceName": "Newtonsoft.Json",
         "typeName": "JsonConverter",
-        "arguments": [ "typeof(EnumConverter)" ]
+        "arguments": [ "typeof(Microsoft.CodeAnalysis.Sarif.Readers.EnumConverter)" ]
```

Comment:
>[ "typeof(Microsoft.CodeAnalysis.Sarif.Readers.EnumConverter)" ] [](start = 21, length = 64)

this was needed bcuz now we have ambiguous reference in the autogenerated\ThreadFlowLocation.cs for "EnumConverter" due to addition of System.ComponentModel namespace.